cmd/utils/flags.go

```diff
@@ -218,7 +218,17 @@ var (
 	}
 	EtherbaseFlag = cli.StringFlag{
 		Name:  "etherbase",
-		Usage: "Public address for transaction broadcasting and block mining rewards (default = first account)",
+		Usage: "Public address for transaction broadcasting and block mining rewards (default = first account, deprecated, use --tx-fee-recipient and --validator instead)",
+		Value: "0",
+	}
+	MinerValidatorFlag = cli.StringFlag{
+		Name:  "miner.validator",
+		Usage: "Public address for participation in consensus (default = first account)",
+		Value: "0",
+	}
+	TxFeeRecipientFlag = cli.StringFlag{
+		Name:  "tx-fee-recipient",
+		Usage: "Public address for block transaction fees (default = first account)",
```

Comment:
This address also receives gateway fees, which should be mentioned in this comment.